Store/StoreBraintreePaymentProvider.php

```diff
@@ -485,7 +485,7 @@ protected function generateExceptionFromResponse(Braintree\Base $response)
 			return $e;
 		}
 
-		return new StorePaymentBraintreeException($response->message);
+		return new StorePaymentBraintreeException(print_r($response, true));
```

Comment:
I'm not sure if we want to use the whole response as the error message, the result I got when testing was pretty long. If we want to include this info I think Sentry has an additional fields section that we might be able to put this in? https://docs.sentry.io/learn/context/

This was the error message I got: 

```
Error creating Braintree customer:
Braintree\Result\Error Object
(
[success] => 
[_attributes:protected] => Array
(
[errors] => Braintree\Error\ErrorCollection Object
(
[_errors:Braintree\Error\ErrorCollection:private] => Braintree\Error\ValidationErrorCollection Object
(
[_errors:Braintree\Error\ValidationErrorCollection:private] => Array
(
)

[_nested:Braintree\Error\ValidationErrorCollection:private] => Array
(
[customer] => Braintree\Error\ValidationErrorCollection Object
(
[_errors:Braintree\Error\ValidationErrorCollection:private] => Array
(
[0] => Braintree\Error\Validation Object
(
[_attribute:Braintree\Error\Validation:private] => email
[_code:Braintree\Error\Validation:private] => 81604
[_message:Braintree\Error\Validation:private] => Email is an invalid format.
)

)

[_nested:Braintree\Error\ValidationErrorCollection:private] => Array
(
[creditCard] => Braintree\Error\ValidationErrorCollection Object
(
[_errors:Braintree\Error\ValidationErrorCollection:private] => Array
(
)

[_nested:Braintree\Error\ValidationErrorCollection:private] => Array
(
[billingAddress] => Braintree\Error\ValidationErrorCollection Object
(
[_errors:Braintree\Error\ValidationErrorCollection:private] => Array
(
)

[_nested:Braintree\Error\ValidationErrorCollection:private] => Array
(
)

[_collection:protected] => Array
(
)

)

)

[_collection:protected] => Array
(
)

)

)

[_collection:protected] => Array
(
)

)

)

[_collection:protected] => Array
(
)

)

)

[params] => Array
(
[customer] => Array
(
[firstName] => Meg
[lastName] => Mitchell
[email] => x@y.z
[creditCard] => Array
(
[cardholderName] => Meg Mitchell
[expirationDate] => 12/18
[options] => Array
(
[makeDefault] => true
)

[billingAddress] => Array
(
[countryCodeAlpha2] => CA
[firstName] => Meg
[locality] => Pender
[postalCode] => V0N2M2
[streetAddress] => 3735 Frigate Rd
[region] => BC
[lastName] => Mitchell
)

[deviceSessionId] => 
[fraudMerchantId] => 
)

)

)

[message] => Email is an invalid format.
[creditCardVerification] => 
[transaction] => 
[subscription] => 
[merchantAccount] => 
[verification] => 
)

)
```